### PR TITLE
Added Realtek v1.0.11 USB 3.0 NIC driver.

### PR DESCRIPTION
**Pull request by joliver**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
It’s missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md) geared for that.

Also, please [squash your commits](http://davidwalsh.name/squash-commits-git).

File: Casks/rtunic.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'rtunic' do
+  version '1.0.11'
+  sha256 'ece924b9e3cabbcf3170529e559af087377ef27fc4106054656e5d6bc3f1a96f'
+
+  url 'https://d2c6jjk3vnoatm.cloudfront.net/spree/products/accessoies/159/DA017E769E0E164_macos0006-RTUNICv1.0.11.zip'
```
Comment (vitorgalvao) [inline Casks/rtunic.rb]: Should be `url "https://d2c6jjk3vnoatm.cloudfront.net/spree/products/accessoies/159/DA017E769E0E164_macos0006-RTUNICv#{version}.zip"`.

Also, please point me to the actual download page so I can verify this URL.

File: Casks/rtunic.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name 'rtunic'
+  homepage 'https://www.anker.com/products/taxons/114/Connectivity'
+  license :gratis
+
+  pkg 'RTUNICv1.0.11/RTUNICv1.0.11.pkg'
```
Comment (vitorgalvao) [inline Casks/rtunic.rb]: `pkg "RTUNICv#{version}/RTUNICv#{version}.pkg"`.

**Comment by joliver** [comment]
This applies to various products they offer, but the one I purchased is here:
https://www.anker.com/support/download?keyword=Aluminum%203-Port%20USB%203.0%20and%20Ethernet%20Hub

Click on the "Upgrade Patch (Mac OS 10.10) and you'll see the URL.

File: Casks/rtunic.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/rtunic.rb]: Thank you. In that case, this needs a `# d2c6jjk3vnoatm.cloudfront.net was verified as official when first introduced to the cask` comment above this line.

**Comment by joliver** [comment]
Call me a total noobie at this, but I'm not quite sure how to do the uninstall.

The dmg that's downloaded has an `uninstall.zip` file and inside is a file called `uninstall.command`. I can't quite figure out how to invoke that script from the `uninstall` stanza.

**Comment by vitorgalvao** [comment]
Since the uninstall script is inside a zip it’s a bit trickier (still doable, though). But the rest of the documentation tells you how to reproduce their actual script. I’ll do it.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in e6122497ca5dc3ff62a692e14e8c315af1c9b56d. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**